data/strategies/StrategyTUSDypool.json

```diff
@@ -2,5 +2,6 @@
   "$schema": "strategy",
   "name": "StrategyTUSDypool",
   "description": "This vault deposits TUSD into the YPool on Curve.fi. The yCRV are then deposited into the Curve YPool yVault.",
```

Comment:
I'd be great to use the same standard description that you've put in others here